packages/fxa-content-server/app/scripts/lib/glean/index.ts

```diff
@@ -79,11 +86,28 @@ const createEventFn = (eventName) => async () => {
     return;
   }
 
-  name.set(eventName);
-  await populateProperties();
+  event.name.set(eventName);
+  await populateMetrics();
   accountsEvents.submit();
 };
 
+const createEventWithPropertiesFn = (eventName) => {
```

Comment:
Really no reason to have two function signatures.  I'm setting this PR back to draft status so I can update all the ping emitting function to take the optional event properties hash.